fr.js

```diff
@@ -744,34 +744,34 @@ windows.playerlist.ban Êtes-vous sûr de vouloir bannir {0} de cette partie?
 windows.playerlist.flag Êtes-vous sûr de vouloir taguer {0}?
 windows.playerlist.unflag Êtes-vous sûr de vouloir détaguer {0}?
 windows.playerlist.punish Êtes-vous sûr de vouloir prendre action sur {0}?
-item.view.open Open Model Viewer
-item.view.more More Information
-item.sell.cant Item cant be sold.
-item.sell.quick Are you sure you want to Quick Sell {0}?
-social.profile.stats Overall Stats
-social.profile.class Class Stats
-social.profile.sales Sales
-social.profile.sales.market Market Sales
-social.profile.skins Skins [{0}]
-social.profile.skins.none No published Skins from this user
-social.profile.games Games [{0}]
-social.profile.games.none No published Games from this user
-social.profile.assets Assets [{0}]
-social.profile.assets.none No published Assets from this user
-social.profile.mods Mods [{0}]
-social.profile.mods.none No published Mods from this user
-social.search Search Results
-social.market.stats.rare Rarest Items
-social.market.stats.circulation In Circulation
```

Comment:
En Circulation

Reply:
Red lines are those that have been changed, they only exist to compare with new lines. Please check green lines to submit corrections